src/components/NewReleaseBanner/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import useTranslation from 'next-translate/useTranslation'
+import Trans from 'next-translate/Trans'
+
+const NewReleaseBanner = () => {
+  const { t } = useTranslation('home')
+  const Header = (props) => <h2 className="new-release-banner__header" {...props} />
```

Comment:
Same as this [comment](https://github.com/Cobda/cobda-web/pull/37/files#r586359391)